apps/trustlab/src/lib/data/pagify/index.js

```diff
@@ -1,12 +1,11 @@
-import helplines from "./helplines";
 import post from "./post";
-import resources from "./resources";
 
 const COLLECTION_BY_SLUG = {
   opportunities: post,
   incubator: post,
-  helplines,
-  resources,
+  helplines: post,
+  resources: post,
+  events: post,
```

Comment:
@kelvinkipruto can we add reports here as well?
Thinking everything here will be treated like a post.